solidity/dashboard/src/services/liquidity-rewards.js

```diff
@@ -140,7 +140,7 @@ class LiquidityRewards {
 
 class UniswapLPRewards extends LiquidityRewards {
   calculateAPY = async (totalSupplyOfLPRewards) => {
-    totalSupplyOfLPRewards = toTokenUnit(totalSupplyOfLPRewards)
+    totalSupplyOfLPRewards = Token.toTokenUnit(totalSupplyOfLPRewards)
```

Comment:
Something is not right here. For the `totalSupplyOfLPRewards` that equals `105552000000000000000000` the `Token.toTokenUnit()` function returns BigNumber that is NaN. Because of that APY for KEEP-only pool is not displayed.

![image](https://user-images.githubusercontent.com/40306834/115866862-9a606880-a43a-11eb-9446-46cf6662b1fe.png)